mappings/xpub.go

```diff
@@ -20,3 +21,18 @@ func MapToXpubContract(xpub *engine.Xpub) *models.Xpub {
 		NextExternalNum: xpub.NextExternalNum,
 	}
 }
+
+// MapToXpubContract will map the xpub model from spv-wallet to the spv-wallet-models contract
+func MapToXpubContract(xpub *engine.Xpub) *response.Xpub {
+	if xpub == nil {
+		return nil
+	}
+
+	return &response.Xpub{
+		Model:           *common.MapToContract(&xpub.Model),
```

Comment:
We should have also common.OldModel and MapToOldContract for it, because the Model struct has snake_case mapping for json

Reply:
changed it
`*common.MapToOldContract`, now returns `*common.OldModel `